Podfile.lock

```diff
@@ -11,15 +11,15 @@ PODS:
     - AppAuth/ExternalUserAgent (= 1.4.0)
   - AppAuth/Core (1.4.0)
   - AppAuth/ExternalUserAgent (1.4.0)
-  - AppCenter (4.2.0):
-    - AppCenter/Analytics (= 4.2.0)
-    - AppCenter/Crashes (= 4.2.0)
-  - AppCenter/Analytics (4.2.0):
+  - AppCenter (4.3.0):
```

Comment:
Are these `Podfile.lock` changes intentional?

Reply:
I just updated the pods, nothing more. Should I revert this?

Reply:
I wouldn't update them if it's not strictly needed for this PR. I know it's just a minor version but we avoid doing so.

Plus, AppCenter is normally handled by the release team.